core/lib/utils/src/convert.rs

```diff
@@ -80,6 +80,13 @@ pub fn be_words_to_bytes(words: &[U256]) -> Vec<u8> {
         .collect()
 }
 
+pub fn hex_string_to_bytes(s: &str) -> Result<Vec<u8>, ParseIntError> {
```

Comment:
Please use the `hex` crate instead; it's already a dependency in `zksync_contracts`.